### PR TITLE
No Nunsense

### PR DESCRIPTION
**Pull request by NPC1314**
## About The Pull Request

(EDITED: Added the suit to chaplain vendor and edited the suit sprite slightly, naming it religious habit instead of nun robe.
On the nun modesty scale of 1 being a streetwalker in a veil and 10 being mother superior  Inviolata the battleship, the old nun suit was a 10, the new outfit pre-edit a 5, and post edit a 7, plus gender neutral so opening up for more characters. Overall a worthy tradeoff.)

Adds new Chaplain outfit to Autodrobe & Chaplain vendor

## Why It's Good For The Game

Alternative drip for chaplains (nuns) that looks better on some character due to
* Slimmer
* More black
* Blacks blue shaded not red so better compliment color for some accessories / skin colors

Why not replace old nun suit? I like the old one too, but its chonky and lots of white areas so don't always fit.
Why autodrobe? Bit more options for religious costumes in there doesn't hurt.

Nunsense
habit
(the lower image is post-edit)

## Changelog
:cl:
image: new chaplain outfit
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
add to chaplain vendor

**Review by Wallemations — APPROVED** [review]

**Comment by BlueMemesauce** [comment]
It could maybe be gender neutral instead of being another nun outfit. that way more people could wear it

**Comment by NPC1314** [comment]
I'm tempted to edit the PR to include both suggestions, the gender neutral one means a bit of wasted effort/granularity which is annoying but the uptick feels like it outweighs the loss. Wish there was a bodytype icon switch function like there is for the alpha mask.

**Review by Ghommie — COMMENTED** [review]
Frankly, the headwear could as well be a reskin of the nun costume hat. About the tunic, it could be treated as a generic cassock so it's ok for it to be "neutral" at the end of the day, though I have a nit on its sprite having a bit of cowl collar rather than an actual clerical one.

**Comment by EricZilla** [comment]
This looks good! Makes me want to add a sexy nun robe called Bad Habit but I'd be tossed in horny jail

**Comment by LukasBeedellCodestuff** [comment]
Habits are meant to be baggy no? The whole thing looks skin tight latex suit

**Comment by BlueMemesauce** [comment]
> Habits are meant to be baggy no? The whole thing looks skin tight latex-y suit very strange choice to change the old one to this

The old one isn't being replaced, it's just adding a new one
Also there is a new edit that made the new one less baggy

**Comment by NPC1314** [comment]
> Frankly, the headwear could as well be a reskin of the nun costume hat. About the tunic, it could be treated as a generic cassock so it's ok for it to be "neutral" at the end of the day, though I have a nit on its sprite having a bit of cowl collar rather than an actual clerical one.

My dude, I could not help read your review out in a "Well akshually" style internal voice, and I admit I found it unreasonably amusing. As for the contents of it, name changes sure, fair point, easily done, see commit.

File: code/modules/jobs/job_types/chaplain/chaplain_costumes.dm [hunk 1]
```diff
@@ -50,6 +50,14 @@
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS|HANDS
 	flags_inv = HIDEJUMPSUIT
 
+/obj/item/clothing/suit/chaplainsuit/habit
+	name = "tunic"
```
Comment (Ghommie) [inline code/modules/jobs/job_types/chaplain/chaplain_costumes.dm:54]:
```suggestion
	name = "religious tunic"
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
> 
> My dude, I could not help read your review out in a "Well akshually" style internal voice.

My man, you're completely right, I pondered too much and got on a bit of a high horse.

**Pull request merged**